rfcs/0043-ckb-softfork-activation/deploymens.md

```diff
@@ -0,0 +1,7 @@
+# Deployments
+---
+
+List of proposed deployments.
+| Name | Bit | Mainnet Start | Mainnet Timeout | Mainnet State | Testnet Start | Testnet Timeout | Testnet State | RFC |
+| ----------- | ---------- | ---------------- | ---------- |  ----------- | ---------- | ---------------- | ---------- | ---------- |
+| Light Client Protocol | 1 | TBD | TBD | TBD | epoch#5346 | epoch#5616 | active since epoch#5711 | [RFC0043](../0044-ckb-light-client/0044-ckb-light-client.md)
```

Comment:
```suggestion
| Light Client Protocol | 1 | TBD | TBD | TBD | epoch#5346 | epoch#5616 | active since epoch#5711 | [RFC0044](../0044-ckb-light-client/0044-ckb-light-client.md)
```